bitnami/customer-billing/kafka-bitnami/upstream/kafka.yaml

```diff
@@ -401,7 +401,7 @@ spec:
       serviceAccountName: kafka-bitnami
       containers:
         - name: kafka
-          image: docker.io/bitnami/kafka:3.4.0-debian-11-r28
+          image: docker.io/bitnami/kafka:3.7.0-debian-12-r7
```

Comment:
On this specific update:
- no changes should be made to this file, as this is the one created by generating from the helm chart.

On the kafka upgrade:
We haven't tested this version yet, and it might need updates in the other places too.
We have a plan to [upgrade to a more recent helm chart](https://linear.app/utilitywarehouse/issue/PLA2-42/kafka-manifests-upgrade-helm-chart-and-kafka-to-latest) that will also update the version, but we don't have a timeline for this yet.